lib/filters/labelify.rb

```diff
@@ -143,7 +156,7 @@ def process_title node
   new_id = node.text
   new_id.downcase!
   new_id.gsub! %r{(?<=[a-zA-Z0-9()])[ \-_/](?=[a-zA-Z0-9()])}, "-"
-  new_id.gsub! %r{[^a-z\-]}, ""
+  # new_id.gsub! %r{[^a-z\-]}, ""
```

Comment:
why remove this?

Reply:
this isn't relevant to this PR, but it was a local change because sections like "Phase 1" and "Phase 2" would get the same id which the compiler doesn't like and then it fails

Reply:
I would have to take a look at it, but if you want to ignore this change for this PR that's fine too